### PR TITLE
fix(release_actions): enable set_key_value to work with .swift files

### PR DESCRIPTION
**Pull request by iartemiev**
We need to be able to increment values in .swift files, as well as in .rb.
E.g., here: https://github.com/aws-amplify/amplify-ios/blob/main/AmplifyPlugins/Core/AWSPluginsCore/ServiceConfiguration/AmplifyAWSServiceConfiguration.swift

Previously, this action would overwrite the value with single quotes `'` hardcoded. 
This PR updates the regex to ignore the quotes and just replace the value inside of them.

```ruby
$AMPLIFY_VERSION = '1.3.3'
# gets replaced with:
$AMPLIFY_VERSION = '1.3.4'
```

```swift
let version = "1.3.3"
// gets replaced with:
let version = "1.3.4"
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by iartemiev** [comment]
> I think this is complex enough that we should write some tests with fixture data. The regular expression is kind of hard to follow. Worked in testing though.

@jpignata Sounds good and agreed. I'll extract the functionality into a helper and will add some coverage for that

**Comment by iartemiev** [comment]
@jpignata updated 😺

**Comment by jpignata** [comment]
LGTM!
